doc/about/api/index.md

```diff
@@ -8,14 +8,14 @@ Sharg adheres to [semantic versioning](https://semver.org) and provides a stable
 one major release unless otherwise noted.
 
 For details regarding API stability, please see the
-[SeqAn documentation on API stability](http://docs.seqan.de/seqan/3-master-user/about_api.html#api_stability).
+[SeqAn documentation on API stability](https://docs.seqan.de/seqan/3-master-user/about_api.html#api_stability).
 
 # ABI stability
 
 There is no ABI stability.
```

Comment:
Maybe make API/ABI bold in the text. Maybe even just the P and B.

Reply:
```suggestion
There is no A**B**I stability.
```